<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSettingsWidgetGeneral</class>
 <widget class="QWidget" name="DatabaseSettingsWidgetGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Database Metadata</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="dbNameLabel">
        <property name="text">
         <string>Database name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dbNameEdit">
        <property name="accessibleName">
         <string>Database name field</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dbDescriptionLabel">
        <property name="text">
         <string>Database description:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dbDescriptionEdit">
        <property name="accessibleName">
         <string>Database description field</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="defaultUsernameLabel">
        <property name="text">
         <string>Default username:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="defaultUsernameEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="accessibleName">
         <string>Default username field</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>History Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="historyMaxItemsCheckBox">
          <property name="toolTip">
           <string>When saving this setting or editing an entry
the oldest history items of an entry will be
removed such that only the specified amount
of entries remain at most.</string>
          </property>
          <property name="text">
           <string>Limit the amount of history items per entry to:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="historyMaxSizeCheckBox">
          <property name="toolTip">
           <string>When saving this setting or editing an entry
the oldest history items of an entry will be
removed such that the remaining history items
add up to the specified amount at most.</string>
          </property>
          <property name="text">
           <string>Limit the total size of history items per entry to:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="historyMaxSizeSpinBox">
          <property name="toolTip">
           <string>Maximum size of history per entry</string>
          </property>
          <property name="accessibleName">
           <string>Maximum size of history per entry</string>
          </property>
          <property name="suffix">
           <string> MiB</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2000000000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="historyMaxItemsSpinBox">
          <property name="toolTip">
           <string>Maximum number of history items per entry</string>
          </property>
          <property name="accessibleName">
           <string>Maximum number of history items per entry</string>
          </property>
          <property name="maximum">
           <number>2000000000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="recycleBinEnabledCheckBox">
          <property name="toolTip">
           <string>Move entries to a recycle bin group
instead of deleting them from the database.
Entries deleted from the recycle bin are
removed from the database.</string>
          </property>
          <property name="text">
           <string>Use recycle bin</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Additional Database Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="compressionCheckbox">
        <property name="text">
         <string>Enable compression (recommended)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="autosaveDelayCheckBox">
          <property name="toolTip">
           <string>Autosave delay since last change</string>
          </property>
          <property name="accessibleName">
           <string>Autosave delay since last change checkbox</string>
          </property>
          <property name="text">
           <string>Autosave delay</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="autosaveDelaySpinBox">
          <property name="toolTip">
           <string>Autosave delay since last change in minutes</string>
          </property>
          <property name="accessibleName">
           <string>Autosave delay since last change in minutes</string>
          </property>
          <property name="suffix">
           <string> min</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>420000000</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
